portfolio/src/main/java/com/google/sps/servlets/RequestServlet.java

```diff
@@ -21,6 +23,17 @@
 public class RequestServlet extends HttpServlet {
 
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        UserService userService = UserServiceFactory.getUserService();
+
+        String authenticationUrl = "";
+        String loggedin = "false";
```

Comment:
Storing "false" and "true" as a string is rarely the best option. In this case, are we using a `String` because a `boolean` couldn't be added to a list of `Object`s? If so, `Boolean` is the thing to use.

(In Java, all primitive types have `Object` wrappers for situations like this, e.g. `Boolean` for `boolean`, `Integer` for `int`, etc.)